Jenkinsfile

```diff
@@ -0,0 +1,38 @@
+pipeline {
+  agent {
+    label 'sbt-slave'
+  }
+
+  environment {
+    // Ensure that build scripts recognise the environment they are running within
+    CI = ''
+    SCALA_VERSION = '2.10.6'
```

Comment:
@fommil We checked that. Jenkins do not add an environment variable specifically for this purpose.